src/main/java/org/swisspush/redisques/RedisQues.java

```diff
@@ -951,103 +973,98 @@ private void consume(final String queue) {
+                                    log.debug("RedisQues Message removed, queue " + queueName + " is ready again");
+                                    myQueues.put(queueName, QueueState.READY);
+                                    // Notify that we are stopped in case it was the last active consumer
                                     if (stoppedHandler != null) {
                                         unregisterConsumers(false);
                                         if (myQueues.isEmpty()) {
                                             stoppedHandler.handle(null);
                                         }
                                     }
                                     // Issue notification to consume next message if any
-                                    String key2 = getQueuesPrefix() + queue;
                                     if (log.isTraceEnabled()) {
-                                        log.trace("RedisQues read queue: " + key);
+                                        log.trace("RedisQues read queue: " + queueKey);
                                     }
-                                    redisClient.llen(key2, answer1 -> {
+                                    redisClient.llen(queueKey, answer1 -> {
                                         if (answer1.succeeded() && answer1.result() != null && answer1.result() > 0) {
-                                            notifyConsumer(queue);
+                                            notifyConsumer(queueName);
                                         }
                                     });
                                 });
                             } else {
                                 // Failed. Message will be kept in queue and retried later
-                                log.debug("RedisQues Processing failed for queue " + queue);
+                                log.debug("RedisQues Processing failed for queue " + queueName);
                                 
                                 // reschedule
-                                log.debug("RedisQues will re-send the message to queue '" + queue + "' in " + retryInterval + " seconds");
-                                rescheduleSendMessageAfterFailure(queue, retryInterval);
+                                log.debug("RedisQues will re-send the message to queue '" + queueName + "' in " + retryInterval + " seconds");
+                                rescheduleSendMessageAfterFailure(queueName, retryInterval);
                             }
                         });
                     } else {
                         // This can happen when requests to consume happen at the same moment the queue is emptied.
-                        log.debug("Got a request to consume from empty queue " + queue);
-                        myQueues.put(queue, QueueState.READY);
+                        log.debug("Got a request to consume from empty queue " + queueName);
```

Comment:
What about
```java
if (log.isDebugEnabled()) {
	//.....
}
```
?

Reply:
Is this your new hobby?

Reply:
Of course :-) I had to find something to look important ;-)